### ISSUE TITLE
I just want to quantify the various cell types within T cells, is this feasible?

### ISSUE THREAD

**Issue by M-jk22** [Danko-Lab/BayesPrism#83]
Hi,
I'm interested in studying T cells specifically. So, I'd like to use the tool BayesPrism to quantify the various types of T cells within the bulk data, including CD8 T cells, CD4 T cells, NK T cells, and Treg cells.

**Comment by tinyi** [comment]
Yes. You may simply specify each individual T cell subtype as an individual
cell type. It would be a good idea to have 50 or marker genes for each cell
type to ensure high signal to noise ratio for the deconvolution and avoid
having cell types being too similar to each other.

Best,

Tinyi

**Comment by guodudou2** [comment]
@tinyi Hi Tinyi, will the marker gene approach work for a reference with only T cell subtypes or still requires comprehensive cell types present in the reference? Thanks!
